src/tribler/gui/network/request.py

```diff
@@ -87,7 +87,7 @@ def update_status(self, status_code: int):
         self.logger.debug(f'Update {self}: {status_code}')
         self.status_code = status_code
 
-    def _on_finished(self):
+    def on_finished(self):
```

Comment:
This change doesn't relate to the solution to the described problem.

Reply:
Maybe make it private?

```python
def __on_finished(self):
```

That way, the reader will instantly know that the method is not a part of the `Request`'s public API.